OhmGraphite/PrometheusConfig.cs

```diff
@@ -1,25 +1,33 @@
-﻿namespace OhmGraphite
+﻿using LibreHardwareMonitor.Hardware.Controller.AeroCool;
```

Comment:
I'm assuming this is an unused import

Reply:
Correct, Visual Studio tried to be smart or something...